src/Configuration/ConfigurationFactory.php

```diff
@@ -113,6 +113,12 @@ public function create(
 
         $testFramework = $testFramework ?? $schema->getTestFramework() ?? TestFrameworkTypes::PHPUNIT;
 
+        [$skipCoverage, $existingCoverageBasePath] = self::retrieveExistingCoverageBasePath(
```

Comment:
IMO, now the method name conflicts with return values. It doesn't return path, but some flag as well.

What about setting it outside the method as:

```php
$skipCoverage = $existingCoverageBasePath !== null;
```

Reply:
👍